include/Core/EmailLogger.php

```diff
@@ -76,18 +82,20 @@ public function log_email( $original_mail_info ) {
 				'message'     => '',
 				'headers'     => '',
 				'attachments' => array(),
+				'ip_address' => $_SERVER['REMOTE_ADDR'],
```

Comment:
This looks good.

You can remove the above commented code. Once the commented out code is removed we are good to go with this issue.